src/Raven.Server/Routing/RouteScanner.cs

```diff
@@ -69,6 +69,7 @@ public class RouteScanner
                                 route.RequiredAuthorization, 
                                 route.SkipUsagesCount,
                                 route.CorsMode,
+                                route.IsDebugInformationEndpoint,
                                 route.IsDebugInformationEndpoint);
```

Comment:
Invalid argument - 2nd one should be `route.DisableOnCpuCreditsExhaustion` instead of `route.IsDebugInformationEndpoint`